website/package.json

```diff
@@ -29,15 +29,23 @@
     "@hashicorp/react-vertical-text-block-list": "2.0.1",
     "algoliasearch": "4.3.0",
     "babel-plugin-import-glob-array": "0.2.0",
+    "dotenv": "^8.2.0",
+    "glob": "^7.1.6",
+    "gray-matter": "^4.0.2",
     "imagemin-mozjpeg": "9.0.0",
     "imagemin-optipng": "8.0.0",
     "imagemin-svgo": "8.0.0",
+    "isomorphic-unfetch": "^3.0.0",
```

Comment:
This isn't needed in nextjs anymore

Reply:
🍒

Reply:
Although if it's being used in a script outside of nextjs, like the search index population, it would be needed. But then we could just use `node-fetch` since it doesn't run on the client.